js/script.js

```diff
@@ -0,0 +1,82 @@
+import { $taskForm, $taskTitle, $taskList, $btnCreate } from '../js/elements.js';
+import { stripSanitizedParts } from '../js/utils/utilities.js';
+
+let tasks = [];
+const $errorMessage = document.createElement("p");
+$errorMessage.className = "error-message";
+
+const openForm = () => {
+    $taskForm.classList.add("show");
+};
+
+const closeForm = () => {
+    $taskForm.classList.remove("show");
+};
+
+const addTask = () => {
+    let title = $taskTitle.value.trim();
+    if (title === "") {
+        showError("Please enter a task title.");
+        return;
+    }
+
+    const displayTitle = stripSanitizedParts(title);
+
+    const newTask = {
+        id: Date.now(), 
+        title: displayTitle,
+        createdAt: new Date().toLocaleDateString(),
```

Comment:
![image](https://github.com/rza-wtag/Todo_App/assets/144016889/8ddfefbd-67d7-42ec-80e8-d3bba56ed00b)

We show date in this format `DD.MM.YY`. So, format `createdAt` according to that.